homeassistant/components/swiss_public_transport/sensor.py

```diff
@@ -101,14 +106,23 @@ class SwissPublicTransportSensor(
 
     _attr_attribution = "Data provided by transport.opendata.ch"
     _attr_icon = "mdi:bus"
+    _attr_has_entity_name = True
+    _attr_translation_key = "departure"
 
     def __init__(
-        self, coordinator: SwissPublicTransportDataUpdateCoordinator, name: str
+        self,
+        coordinator: SwissPublicTransportDataUpdateCoordinator,
+        unique_id: str,
     ) -> None:
         """Initialize the sensor."""
         super().__init__(coordinator)
         self._coordinator = coordinator
```

Comment:
```suggestion
```

Reply:
Thanks, this is indeed redundant as the coordinator does it already himself :)

Reply:
CoordinatorEntity*